src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/model/IndividualRiskScores.kt

```diff
@@ -32,4 +32,13 @@ data class IndividualRiskScores(
 
   @Schema(example = "LOW", description = "")
   @get:JsonProperty("sara") val sara: String? = null,
+
+  @Schema(example = "LOW")
+  @get:JsonProperty("saraRiskOfViolenceTowardsPartner") val saraRiskOfViolenceTowardsPartner: String? = null,
+
+  @Schema(example = "LOW")
+  @get:JsonProperty("saraRiskOfViolenceTowardsOthers") val saraRiskOfViolenceTowardsOthers: String? = null,
```

Comment:
I think we still need to report sara outcome

We need to take whatever is the higher of the 2 saras
Something like:
sara {
  result: "HIGH",
  saraRiskOfViolenceTowardsPartner: "MEDIUM",
  saraRiskOfViolenceTowardsOthers: "HIGH"
}

Reply:
Good point! Will address now.